benchmarks/lucet-benchmarks/src/seq.rs

```diff
@@ -259,10 +259,10 @@ fn run_hello<R: RegionCreate + 'static>(c: &mut Criterion) {
     c.bench_function(&format!("run_hello ({})", R::TYPE_NAME), move |b| {
         b.iter_batched_ref(
             || {
-                let null = std::fs::File::open("/dev/null").unwrap();
                 let ctx = WasiCtxBuilder::new()
-                    .args(&["hello"])
-                    .fd(1, null)
+                    .expect("create a new WASI context")
+                    .args(["hello"].iter())
+                    .expect("WASI arguments")
```

Comment:
...and furthermore it should defer all the errors to `.build()`. Maybe I'll do a PR when I have a chance